Add awsfargate module to collect container logs from Amazon ECS on Fargate (#25041)
elastic/beats: filebeat/docs/modules/awsfargate.asciidoc

////
This file is generated! See scripts/docs_collector.py
////

[[filebeat-module-awsfargate]]
[role="xpack"]

:libbeat-xpack-dir: ../../../x-pack/libbeat

:modulename: awsfargate
:has-dashboards: false

== AWS Fargate module

beta[]

> This is a module for collect container logs from Amazon ECS on Fargate.

Reviewer: ```suggestion
This module can be used to collect container logs from Amazon ECS on Fargate.
```

There is another place to apply this change (in case you agree) too, so please don't rely just on this GH `commit`.

Author: Will do :) Thanks!
